assets/js/page-enhancements.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Script file to apply progressive enhancement techniques to page elements.
+ * Any code here should be used only to enhance the page and UX if the user has JS enabled.
+ *
+ * If the user does not have JS enabled, or the code within this script does not run for some reason
+ * the application should fall back to standard HTML functionality and should continue to operate.
+ *
+ * No code should be written here that renders a page non-functional if JS is not enabled ot the script does not run.
+ */
+
+;(autoFocusBarcodeField = ($, document) => {
```

Comment:
This is known as an IIFE (immediately invoked function expression) which internally uses jquery to focus the fields. Using a named function as the IIFE means (hopefully) that its self documenting

Reply:
I'm familiar with IIFE but not with that syntax (semi-colon first). 🤷‍♂️

Reply:
The semicolon as added by prettier tbh because it had some weird objection to the opening bracket !!
Hey ho 🤷‍♂️